src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimContinuingOnlineRespondentPartyForSpecNotificationHandler.java

```diff
@@ -72,7 +72,7 @@ private CallbackResponse notifyRespondentForClaimContinuingOnline(CallbackParams
             generatePIPEmail(caseData);
         }
 
-        generatePIPLetter(callbackParams);
+//        generatePIPLetter(callbackParams);
```

Comment:
Is this an intentional change?

Reply:
oops it will introduce a bug if merged to master!! this code generates letter and sends to bulk printing to be posted to lip defendant as a letter with pin and case number and instructions on what to do

Reply:
Not, not intentional at all. I'll undo this